lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -233,3 +234,62 @@ function dosomething_kudos_get_kudos_for_user_by_file($fid) {
 
   return $kudos;
 }
+
+/**
+ * Check to see if the kudos term has been selected by the user on specified
+ * reportback.
+ *
+ * @param  [type] $id
+ * @param  object $reportbackItem
+ * @return bool
+ */
+function dosomething_kudos_term_is_selected($id, $reportbackItem) {
+  if (! $reportbackItem->existing_kids) {
+    return false;
+  }
+
+  if (! isset($reportbackItem->existing_kids[$id])) {
+    return false;
+  }
+
+  return true;
+}
+
+/**
+ * Get the icon associated with the kudos term id.
+ *
+ * @param  string $id
+ * @return string
+ * @TODO: this will eventually be changed to refere to SVG icon paths for our custom kudos icons!
+ */
+function dosomething_kudos_get_icon_by_term_id($id) {
+  settype($id, 'string');
+
+  $icons = [
+    '641' => '&#128150;',
```

Comment:
these `TIDs` (since currently we are only using one sitewide) should probably be defined at the top of the file as a constant. I would imagine once we start having overrides per campaign, we will want to pull this from a global variable, but we can just hardcode the one for now.